docs/platform/concepts/carbon-footprint.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Carbon footprint
+early: true
+---
+
+import EarlyBadge from "@site/src/components/Badges/EarlyBadge";
+
+You must be a [super admin](/docs/platform/howto/make-super-admin) to access this feature.
+
+The carbon footprint page shows estimates of the greenhouse gas emissions associated with service usage in your organization.
+
+Emissions are measured in metric tons of carbon dioxide-equivalent, which includes
+multiple greenhouse gasses. The following information is shown on the dashboard:
+
+- Estimated emissions this month: Shows an estimate of emissions for the current calendar
```

Comment:
These 3 points are a bit confusing... We show the emissions for the last 3, 6 and 12 months for a specified project on a per month basis. Soon we will also show the total emissions for the selected project and timeframe.

Reply:
There are also the estimated emissions shown at the top of the page. The items in this list reflect the titles in the UI.